### PR TITLE
Storing Job Applications on the Backend

### PR DESCRIPTION
**Pull request by julianjear**
- Created a new table on the backend to store job applications.
- Added a JobApplication type to the schema and coded the resolvers and models logic so that we can query the data.

### CODE REVIEW

File: postgres/migrations/V1.27__job_applications_table.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-- Job Applications Table
+DROP TABLE IF EXISTS job_applications;
+CREATE TABLE job_applications (
+	application_id serial PRIMARY KEY,
+	full_name VARCHAR(40) NOT NULL,
+    phone_number VARCHAR(15) NOT NULL,
+	email text NOT NULL,
+	cover_letter text,
+	jobadid INTEGER,
```
Comment (julianjear) [inline postgres/migrations/V1.27__job_applications_table.sql:9]: I wasn't sure whether I should also include the job title and company name on top of the id values for the jobad and the company but I felt that if we have the id we can retrieve the other values we need.

File: web-app/src/translations/en.tsx [hunk 2]
```diff
@@ -515,7 +515,7 @@ export default {
 				"Learn how you can lower turnover rates, decrease recruiting costs, improve your recruiting practices, and much more with our resources",
 			resources: "Resources",
 			headerText:
-				"Recruit and retain the best workforce in Tijuna with Vize",
+				"Recruit and retain the best workforce in Tijuana with Vize",
```
Comment (julianjear) [inline web-app/src/translations/en.tsx:518]: Was too lazy to create a separate branch for a single word spelling correction

File: codegen.yml [hunk 3]
```diff
@@ -16,6 +16,7 @@ generates:
         StarRatings: api-server/src/models/types#StarRatings
         Company: api-server/src/models/types#Company
         JobAd: api-server/src/models/types#JobAd
+        JobApplication: api-server/src/models/types#JobApplication
```
Comment (julianjear) [inline codegen.yml:19]: For some reason, I had to manually add this line of code into the codegen.yml file. Is this supposed to be done manually?

File: postgres/migrations/V1.27__job_applications_table.sql (hunk 1 above)
Comment (joshuatshaffer) [reply postgres/migrations/V1.27__job_applications_table.sql:9]: That's correct. The way you've done this is called "normalized" and it's usually considered the best way to do things.

File: codegen.yml (hunk 3 above)
Comment (joshuatshaffer) [reply codegen.yml:19]: No, it's a manual thing unfortunately.

File: api-server/src/graphql/schema.graphql [hunk 4]
```diff
@@ -394,6 +391,17 @@ type JobAd {
 	company: Company!
 }
 
+type JobApplication {
```
Comment (joshuatshaffer) [inline api-server/src/graphql/schema.graphql:394]: Should we return this on the public API? This is private information. It at least needs a permissions check.

File: web-app/src/translations/en.tsx (hunk 2 above)
Comment (joshuatshaffer) [reply web-app/src/translations/en.tsx:518]: That's fine. It's a small change.

**Review by joshuatshaffer — CHANGES_REQUESTED** [review]

File: web-app/src/translations/en.tsx (hunk 2 above)
Comment (joshuatshaffer) [reply web-app/src/translations/en.tsx:518]: That's fine. It's a small change.

File: postgres/migrations/V1.27__job_applications_table.sql (hunk 1 above)
Comment (joshuatshaffer) [reply postgres/migrations/V1.27__job_applications_table.sql:9]: That's correct. The way you've done this is called "normalized" and it's usually considered the best way to do things.

File: codegen.yml (hunk 3 above)
Comment (joshuatshaffer) [reply codegen.yml:19]: No, it's a manual thing unfortunately.

File: api-server/src/graphql/schema.graphql (hunk 4 above)
Comment (julianjear) [reply api-server/src/graphql/schema.graphql:394]: Went ahead and added a companyId check to make sure the right company is querying the data.

**Review by joshuatshaffer — APPROVED** [review]

**Pull request merged**